editor/src/clj/editor/image.clj

```diff
@@ -25,13 +26,18 @@
             [editor.workspace :as workspace])
   (:import [com.dynamo.bob.pipeline TextureGenerator$GenerateResult]
            [com.dynamo.bob.textureset TextureSetGenerator$UVTransform]
-           [com.dynamo.bob.util TextureUtil]
            [java.awt.image BufferedImage]))
 
 (set! *warn-on-reflection* true)
 
 (def exts ["jpg" "png"])
 
+(defn image-path?
+  [path]
+  (->> exts
+       (some (partial str/ends-with? (str/lower-case path)))
+       boolean))
+
```

Comment:
Moved to `image` ns and also added `str/lower-case` to handle some additional missed cases.